### DISCUSSION TITLE
Hexo Icarus Page Loading is Considered Slow by Google.

### DISCUSSION THREAD

**Discussion by leimao** [ppoffice/hexo-theme-icarus#1000] [General]
Hexo Icarus is page loading speed too low. Recently I got "warned" by the Google Search Console that my page loading speed is sub-optimum. They provided me a tool to check the page loading speed. So I checked a couple of pages from my site and the Icarus demo pages.

Icarus demo page:
https://pagespeed.web.dev/report?url=https%3A%2F%2Fppoffice.github.io%2Fhexo-theme-icarus%2FConfiguration%2Ficarus-user-guide-configuring-the-theme%2F

My page:
https://pagespeed.web.dev/report?utm_source=psi&utm_medium=redirect&url=https%3A%2F%2Fleimao.github.io%2Fblog%2FTmux-Tutorial%2F&hl=en

It seems that the loading speed for both pages are marked very slow. Is there any way to optimize this? Is it because I added too many Adsense? Thank you.

**Comment by ppoffice** [comment]
Disable unnecessary third party libraries/plugins could speed up the page load. You can also disable animations to make the page interactive to users at an early time.
